### DISCUSSION TITLE
SSL Certificate Limit Increase - 20 InternalZoneSSLCert limit / 20 ExternalZoneSSLCertLocalIfEast limit

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#3039] [General Questions & Discussion] [answered]
Just found out that the default aws setup for Hubs Personal give a total of 20  "Internal Zone SSL Certificates". This means if you do more than 20 installations of hubs you get an error and have to ask for a limit increase:

https://forums.aws.amazon.com/thread.jspa?threadID=238632

It took them 3 days to do this for us which meant a loss of service. If you think that you will do a lot of installations and want to keep your system running, its probably best to ask for this limit increase in advance,

I hope this saves someone the headache we have had to endure!

**Comment by robinkwilson** [comment, marked as answer]
Excellent link! Thanks for posting that here. You pretty much answered your question.

Though you'll definitely need to ask for a certificate limit increase for less than 20 installations. [Hubs Docs: ExternalZoneSSLCertLocalIfEast or InternalZoneSSLCert Solution 3](https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#potential-solutions)

1 Hubs installation makes multiple SSL Certs to authenticate your urls. 

My first deploy, when I changed nameservers to NOT point to aws (don't do this! Keep them on the aws defaults), I did 3-6 Hubs Cloud deploys and likely hit the SSL cert limit, though I'm not sure which number deploy it was.

**Reply by ocularvr** [reply]
Thanks! Yeah this link will save you two weeks of downtime if you need to reinstall and have the default 20 limit. I hope it saves people head aches!
